Firestore/core/src/firebase/firestore/util/async_queue.h

```diff
@@ -89,8 +89,30 @@ class AsyncQueue : public std::enable_shared_from_this<AsyncQueue> {
   using Operation = Executor::Operation;
   using Milliseconds = Executor::Milliseconds;
 
+  enum class Mode {
+    /**
+     * The default mode of an AsyncQueue after creation. All tasks are allowed.
```

Comment:
Ultranit: can you add backticks around `AsyncQueue` and `Mode::Restricted`? (here and below)